packages/psss-server/src/routes/alerts/FetchAlertsRoute.ts

```diff
@@ -21,22 +27,46 @@ function validateStatus(category: string): asserts category is AlertCategory {
 
 export class FetchAlertsRoute extends Route {
   async buildResponse() {
-    const { startWeek, endWeek, orgUnitCodes } = this.req.query;
-    const { category } = this.req.params;
-
-    validateStatus(category);
-    const reportCode = CATEGORY_TO_REPORT[category];
+    const { startWeek, endWeek } = this.req.query;
 
-    const reportData = await this.reportConnection?.fetchReport(
-      reportCode,
-      orgUnitCodes?.split(','),
-      [startWeek, endWeek],
+    const alertData: AlertReportRow[] = await this.fetchAlertData();
+    const data = await Promise.all(
+      alertData.map(async alert => {
+        const confirmedData = await this.fetchConfirmedDataForAlert(alert);
+        return { ...alert, ...confirmedData };
+      }),
     );
 
     return {
       startWeek,
       endWeek,
-      data: reportData,
+      data,
     };
   }
+
+  private fetchAlertData = async () => {
+    const { startWeek, endWeek, orgUnitCodes } = this.req.query;
+    const { category } = this.req.params;
+
+    validateStatus(category);
+    const alertReport = CATEGORY_TO_REPORT[category];
+    const { results } = await this.reportConnection.fetchReport(
+      alertReport,
+      orgUnitCodes?.split(','),
+      [startWeek, endWeek],
+    );
+
+    return results as AlertReportRow[];
+  };
+
+  private fetchConfirmedDataForAlert = async (alert: AlertReportRow) => {
+    const { organisationUnit, period, syndrome } = alert;
+    const { results } = await this.reportConnection.fetchReport(
+      `PSSS_${syndrome}_Confirmed_Report`,
+      [organisationUnit],
+      [period],
```

Comment:
Sorry for snooping, but I've just played around with latest PSSS version a bit, shouldn't this be
```suggestion
      [period, endWeek],
```
So that the `Cases since alert began` can be cumulative from the alert triggered date (period) until the selected week (endWeek)? Otherwise I think it will just return weekly data

Reply:
Hey Chris, great catch! 👀  What you say makes sense, I'll change this

Reply:
I'll merge this as is for now, then investigate its behaviour when the UI is also in